src/utils.ts

```diff
@@ -25,3 +26,22 @@ export function normalizeHexAddress(address: string | Buffer): string {
   const even = noPrefix.length % 2 === 0 ? noPrefix : `0${noPrefix}`
   return `0x${Buffer.from(even, "hex").toString("hex")}`
 }
+
+export function toChecksumAddress(address: string): string {
+  const addressWithOutPrefix = address.toLowerCase().replace("0x", "")
```

Comment:
We should probably go ahead and do full normalization with `normalizeHexAddress` while we're at it here.